README.md

```diff
@@ -19,16 +19,64 @@ Using yarn:
 yarn add @solana/web3.js @project-serum/serum
 ```
 
-## Usage
+# Guide
+
+This guide will cover every aspect of Serum-JS and how it can be used to interact with the Serum DEX. For a technical introduction to the Serum DEX, please take a look at: https://docs.google.com/document/d/1isGJES4jzQutI0GtQGuqtrBUqeHxl_xJNXdtOv4SdII
+
+To interact with the DEX, a Wallet adapter is needed: https://github.com/project-serum/sol-wallet-adapter.
```

Comment:
This is only true for integrating the dex into a web page. The typical user would use an `Account` directly.